src/fsharp/MethodOverrides.fs

```diff
@@ -308,28 +324,28 @@ module DispatchSlotChecking =
                                 // Error will be reported below in CheckOverridesAreAllUsedOnce 
                                 ()
 
-                            elif argTys.Length <> vargtys.Length then 
-                                fail(Error(FSComp.SR.typrelMemberDoesNotHaveCorrectNumberOfArguments(FormatOverride denv overrideBy, FormatMethInfoSig g amap m denv dispatchSlot), overrideBy.Range))
+                            elif not (listSameLength argTys vargtys) then 
+                                fail(Error(FSComp.SR.typrelMemberDoesNotHaveCorrectNumberOfArguments(FormatOverride denv overrideBy, formatMethodInfoSignature dispatchSlot), overrideBy.Range))
                             elif mtps.Length <> fvmtps.Length then
```

Comment:
If `mtps` is a list, use `listSameLength` here as well.

Reply:
I think there is already an issue/suggestion. It would be easy to implement, the hard part is to decide how to represent it on the tooltip.

Reply:
#3110 revolves around similar topic, in this case it would display:

![image](https://user-images.githubusercontent.com/87944/40535525-37a5d090-5fbe-11e8-9e98-0a3bbfe8ece2.png)
> Typars = Typar list